deployments/infrastructure-env-production.nix

```diff
@@ -13,7 +13,7 @@ in {
       ./../modules/datadog.nix
     ];
 
-    services.dd-agent.tags = ["env:production"];
+    services.dd-agent.tags = ["env:production", "depl:${config.deployment.name}"];
```

Comment:
It would be nice to have a tag that distinguishes mainnet nodes from production-grade infrastructure nodes. Perhaps `env:production-infra` would be good.

Reply:
If we do that, we'd need to be sure all the declaratively specified DD monitors are created for both envs.

Reply:
@jmitchell, we can do select those by intersectingf:
- `env: production` 
- `depl: infra`

Reply:
Ah, yes. They're separate deployments. 👍